src/Tribe/Assets_Pipeline.php

```diff
@@ -35,9 +35,9 @@ public function prevent_underscore_conflict( $tag, $handle ) {
 
 		if ( 'underscore' === $handle ) {
 			$dir = Tribe__Main::instance()->plugin_url . 'src/resources/js';
-			$tag = "<script src='{$dir}/underscore-before.js'></script>\n"
+			$tag = "<script src='{$dir}/underscore-before.js' defer></script>\n"
```

Comment:
Should we be hard code `defer`ing this script? What was the idea here? Want to make sure we considered unintended consequences.

Reply:
Ah, good question!

My goal here is make sure all three of these scripts execute in the correct order AND are deferred. The before and after scripts can be deferred as long as underscore is also deferred, however it is possible it won't be - if it has a non deferrable dependency. 

I think this can be adjusted to take that into account, possibly directly here with a check, or possible all three scripts (underscore-before, underscore and underscore-after) should all use the `wp_enqueue_script` approach rather than directly outputting these wrapper scripts. 

I will review the scripts again and suggest an improved approach here.

Reply:
@stratease - I improved this in a0b38d52b56199f6cd93aa4bc1abd63719d4dc14 - now the defer attributes are only added when `underscrore` is already deferred. I also added an incline comment to explain why the defer attribute is added here (namely: to ensure execution order remains intact, which is true when all scripts are deferred).

Let me know if you have any questions!